build.go

```diff
@@ -1,16 +1,51 @@
 package thin
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
+
 	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/fs"
 	"github.com/paketo-buildpacks/packit/v2/scribe"
 )
 
 func Build(logger scribe.Emitter) packit.BuildFunc {
 	return func(context packit.BuildContext) (packit.BuildResult, error) {
 		logger.Title("%s %s", context.BuildpackInfo.Name, context.BuildpackInfo.Version)
 
+		thinConfigFilepath := filepath.Join(context.WorkingDir, "thin.yml")
+
+		envVarThinConfigFilepath := os.Getenv("BP_THIN_CONFIG_LOCATION")
+		if envVarThinConfigFilepath != "" {
+			if !filepath.IsAbs(envVarThinConfigFilepath) {
+				envVarThinConfigFilepath = filepath.Join(context.WorkingDir, envVarThinConfigFilepath)
+			}
+
+			envVarThinConfigFilepathExists, err := fs.Exists(envVarThinConfigFilepath)
+			if err != nil {
+				return packit.BuildResult{}, err
+			}
+
+			if !envVarThinConfigFilepathExists {
+				return packit.BuildResult{}, packit.Fail.WithMessage("thin config file does not exist at: %s", envVarThinConfigFilepath)
+			}
+
+			thinConfigFilepath = envVarThinConfigFilepath
+		}
```

Comment:
nitpick, i think this could be simplified slightly:
```suggestion
               thinConfigFilepath := os.Getenv("BP_THIN_CONFIG_LOCATION")
                if thinConfigFilepath != "" {
                        if !filepath.IsAbs(thinConfigFilepath) {
                                thinConfigFilepath = filepath.Join(context.WorkingDir, thinConfigFilepath)
                        }

                        thinConfigFilepathExists, err := fs.Exists(thinConfigFilepath)
                        if err != nil {
                                return packit.BuildResult{}, err
                        }

                        if !thinConfigFilepathExists {
                                return packit.BuildResult{}, packit.Fail.WithMessage("thin config file does not exist at: %s", thinConfigFilepath)
                        }
                } else {
                        thinConfigFilepath = filepath.Join(context.WorkingDir, "thin.yml")
                }
```

Reply:
Sure, I'm happy with this change!